### PR TITLE
Clobber all files in generated directory

### PR DESCRIPTION
**Pull request by rupurt**

### CODE REVIEW

File: lib/generators/react_on_rails/templates/base/base/lib/tasks/assets.rake [hunk 1]
```diff
@@ -19,8 +19,6 @@ namespace :assets do
   end
 
   task :clobber do
-    rm_rf "#{Rails.application.config.root}/app/assets/javascripts/generated/vendor-bundle.js"
-    rm_rf "#{Rails.application.config.root}/app/assets/javascripts/generated/client-bundle.js"
-    rm_rf "#{Rails.application.config.root}/app/assets/javascripts/generated/server-bundle.js"
+    rm_rf Dir.glob(Rails.root.join("app/assets/javascripts/generated/*"))
```
Comment (justin808) [inline lib/generators/react_on_rails/templates/base/base/lib/tasks/assets.rake]: :+1:

Comment (justin808) [inline lib/generators/react_on_rails/templates/base/base/lib/tasks/assets.rake]: @rupurt while we're at it, let's change to `rm_r` if that's available for Rake.

no need for rm -rf, right?

Comment (rupurt) [inline lib/generators/react_on_rails/templates/base/base/lib/tasks/assets.rake]: :+1: can't see why not. Do you want me to make it 2nd commit or amend the previous?

**Comment by justin808** [comment]
@rupurt Please rebase on top of master. This will go out in 1.0.2.

**Comment by rupurt** [comment]
@justin808 done

**Pull request merged**